recipes-bsp/efibootguard/efibootguard_0.9.bb

```diff
@@ -12,12 +12,17 @@
 LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
-SRC_URI = "git://github.com/siemens/efibootguard.git;protocol=https;branch=master"
-SRCREV = "ac1685aea75fb3e3d16c0c0e4f8261a2edb63536"
+SRC_URI = "git://github.com/siemens/efibootguard.git;protocol=https;branch=master \
+    file://0001-Makefile-Fix-static-tool-build-on-incremental-builds.patch \
+    file://0002-Makefile-Fix-static-tool-build-on-incremental-builds.patch \
+    file://0003-Find-pkg-config-on-cross-compilation.patch \
+    file://0004-configure-libpci-detection-optional.patch \
```

Comment:
I think I can rather organize another release with all needed patches included. But the last one still needs to some round(s).

Reply:
The only little detail which needs to be solved with efibootguard to keep meta-efibootguard working is https://github.com/siemens/efibootguard/pull/5. But updating meta-efibootguard to the next efibootguard release would be better than cherry-picking many patches from a future release.